src/ui/common/src/components/workflows/nodes/NoneArtifactNode.tsx

```diff
@@ -0,0 +1,23 @@
+import { faChartLine } from '@fortawesome/free-solid-svg-icons';
+import React, { memo } from 'react';
+
+import { ReactFlowNodeData } from '../../../utils/reactflow';
+import Node from './Node';
+
+type Props = {
+  data: ReactFlowNodeData;
+  isConnectable: boolean;
+};
+
+const NoneArtifactNode: React.FC<Props> = ({ data, isConnectable }) => {
+  return (
+    <Node
+      icon={faChartLine}
```

Comment:
@vsreekanti is this the icon that we wish to use?

Perhaps we can use some sort of "function" or "computation" icon here?